README.md

```diff
@@ -56,6 +56,10 @@ Windows is not tested and not supported for now.
 
 - Create a local virtual environment with `virtualenv .` and activate this with `source bin/activate`
 - Install the required dependencies with `pip install -r requirements.txt`
+- If `scancode-toolkit==2.0.1` requirements gives an error
```

Comment:
May be the requirements need to be updated instead?2.0.1 is an old version now.

Reply:
ok, I'll make it 2.2.1!!